packages/concerto-cto/lib/parser.pegjs

```diff
@@ -97,6 +97,10 @@
       namespace: split.join('.')
     };
   }
+  function isPrimitiveType(typeName) {
+        const primitiveTypes = ['Boolean', 'String', 'DateTime', 'Double', 'Integer', 'Long'];
+        return (primitiveTypes.indexOf(typeName) >= 0);
+  }
```

Comment:
nitpick: I guess there isn't a formatter for pegjs files, but could you align the indentation amount with the rest of the file?
```suggestion
  function isPrimitiveType(typeName) {
    const primitiveTypes = ['Boolean', 'String', 'DateTime', 'Double', 'Integer', 'Long'];
    return (primitiveTypes.indexOf(typeName) >= 0);
  }
```